src/common/constants/providers.ts

```diff
@@ -14,6 +14,10 @@ export const SUBSCRIPTION_PROFILE_VERIFIED_CREDENTIAL =
   'alkemio-subscriptions-profile-verified-credential';
 export const SUBSCRIPTION_CALLOUT_MESSAGE_CREATED =
   'alkemio-subscriptions-callout-message-created';
+export const SUBSCRIPTION_CHALLENGE_OPPORTUNITY_CREATED =
+  'alkemio-subscriptions-challenge-opportunity-created';
+export const SUBSCRIPTION_HUB_CHALLENGE_CREATED =
+  'alkemio-subscriptions-hub-challenge-created';
 export const NOTIFICATIONS_SERVICE = 'alkemio-notifications';
 export const WALLET_MANAGEMENT_SERVICE = 'alkemio-wallet-manager';
 export const REDIS_LOCK_SERVICE = 'redis-lock-service';
```

Comment:
any reason to include "challenge" and "hub" in the names?

Reply:
No reason, removing the prefixes.